src/css/main.css

```diff
@@ -153,3 +153,17 @@ nav.menu .menu__list-item--collapsed .menu__link--sublist:after,
     color: #98a3ff;
   }
 }
+
+/* Custom styling for code blocks */
+.theme-code-block {
+  overflow-x: auto; 
+  max-width: 100%; 
+  width: 100%; 
+  box-sizing: border-box;
+}
+/* Custom styling for code blocks content */
+.theme-code-block pre {
+  white-space: pre-wrap; 
+  word-wrap: break-word; 
```

Comment:
Do we have to have this? Wrapping lines can make a big difference to code blocks. I'd rather them just scroll horizontally.

Reply:
```suggestion
```